lib/functions-filters.php

```diff
@@ -10,4 +10,22 @@ function igv_outofstock_text( $availability, $_product ) {
     $availability['availability'] = __('Sold', 'woocommerce');
   }
   return $availability;
-}
\ No newline at end of file
+}
+
+// exclude most recent editorial fron archive
+function exclude_recent_editorial( $query ) {
+  if ( is_admin() || ! $query->is_main_query() )
+    return;
+
+  if ( $query->is_post_type_archive('editorial') ) {
+    $args = array(
+      'post_type' => 'editorial',
+      'post_per_page' => 1,
+    );
+    $recent_editorial = get_posts($args);
+    $recent_id = $recent_editorial[0]->ID;
+    
+    $query->set( 'post__not_in', array( $recent_id ) );
```

Comment:
could possibly use offset here: https://codex.wordpress.org/Class_Reference/WP_Query#Pagination_Parameters but will the archive paginate? I guess yes? Not sure how that works in the design tho